src/libstore/machines.cc

```diff
@@ -170,7 +170,7 @@ static Machine parseBuilderLine(const std::string & line)
 
     return {
         tokens[0],
```

Comment:
Maybe this is a bit specific to me but I had some problems at the start where it took some time to realize which fields mean what, especially because `token`, `tokenizeString` etc. seem very abstract at first.

It may be helpful to include that in a comment, so you don't have to jump back and forth between `Machine` and this method.

Reply:
That makes sense. I do wish we could use `.field = ...` notation for this. (I guess we can soon with a newer C++.) I can add some comments for now. I will do that in a follow-up PR.